data/language/en-GB.txt

```diff
@@ -1579,7 +1579,7 @@ STR_2217    :{WINDOW_COLOUR_2}{COMMA16}°F
 STR_2218    :{RED}{STRINGID} on {STRINGID} hasn’t returned to the {STRINGID} yet!{NEWLINE}Check whether it is stuck or has stalled
 STR_2219    :{RED}{COMMA16} people have died in an accident on {STRINGID}
 STR_2220    :{WINDOW_COLOUR_2}Park Rating: {BLACK}{COMMA16}
-STR_2221    :Park Rating: {COMMA16}
+STR_2221    :Park Rating: {COMMA16},  Company Value: {CURRENCY}
```

Comment:
```suggestion
STR_2221    :Park Rating: {COMMA16}{NEWLINE}Company Value: {CURRENCY}
```
I think this might look better, please test and provide a screenshot

Reply:
<img width="144" alt="Screenshot 2024-02-26 at 09 32 15" src="https://github.com/OpenRCT2/OpenRCT2/assets/63599884/3a6f6594-ed6f-4fc0-9b4e-1466b72bc568">

Reply:
Weird how it broke down things differently, but I still think I prefer it. Will have to see what others think